sky/skylet/job_lib.py

```diff
@@ -41,8 +41,15 @@ def __lt__(self, other):
 
 
 _RAY_TO_JOB_STATUS_MAP = {
-    'PENDING': JobStatus.PENDING,
-    'RUNNING': JobStatus.RUNNING,
+    # These are intentionally set to one status before, because:
+    # 1. when the ray status indicates the job is PENDING the generated
+    # python program should be not started yet, i.e. the job should be INIT.
```

Comment:
1) "Should not be started"
2) What is Sky cached job status?
3) Is it possible to give these Strings a different name? It is easy to confuse `Pending` and Jobstatus's Pending.

Reply:
Fixed the issues! Thanks! The strings represents the output of `ray job status` we cannot directly change them, unless patching the ray code.

Reply:
Ok I think file a Git issue on Ray for the lower case status.

Reply:
That is no longer a problem in the newer version of ray, such as 1.13. ; ) You will see them all in upper case in #969

Reply:
Sound good!